src/binding/c/coll_api.txt

```diff
@@ -0,0 +1,253 @@
+# vim: set ft=c:
+
+MPI_Allgather:
```

Comment:
I don't see MPI_ERR_TYPE, MPI_ERR_BUFFER, MPI_ERR_COUNT, or MPI_ERR_OP for any of the generated collective validation.

Reply:
Added commit "python: add missing error codes from user buffer validation" that fixes this.